spec/board_spec.rb

```diff
@@ -2,78 +2,56 @@
+      expect(board[0]).to eql(1)
+      expect(board[13]).to eql(nil)
     end
   end
-
-  describe 'update_board' do
-    it 'should update position 1 with X' do
-      tictactoe.update_board(1, 'X')
-      expect(tictactoe.board[0]).to eq('X')
+  describe '#update_board' do
+    it 'should be able to update value with chosen position' do
+      tictactoe.update_board(4, 'x')
+      expect(tictactoe.board[3]).to eql('x')
     end
   end
-
-  describe 'won?' do
-    it 'should return true if a player match 3 rows' do
-      tictactoe.update_board(1, 'X')
-      tictactoe.update_board(2, 'X')
-      tictactoe.update_board(3, 'X')
-      expect(tictactoe.won?).to eq(true)
+  describe '#won?' do
+    it 'should return true if either player matches the win criteria' do
+      tictactoe.update_board(4, 'o')
+      tictactoe.update_board(5, 'o')
+      tictactoe.update_board(6, 'o')
+      expect(tictactoe.won?).to eql(true)
     end
-
-    it 'should return true if a player match 3 columns' do
-      tictactoe.update_board(0, 'X')
-      tictactoe.update_board(3, 'X')
-      tictactoe.update_board(6, 'X')
-      expect(tictactoe.won?).to eq(true)
-    end
-
-    it 'should return true if a player match 2 diagonals' do
+  end
+  describe '#full?' do
+    it 'returns false if the board is not full' do
       tictactoe.update_board(1, 'X')
-      tictactoe.update_board(5, 'X')
-      tictactoe.update_board(9, 'X')
-      expect(tictactoe.won?).to eq(true)
+      expect(tictactoe.full?).to eql(false)
     end
-
-    it 'should return false if the game is a draw' do
+    it 'returns true if the board is full' do
       tictactoe.update_board(1, 'X')
       tictactoe.update_board(2, 'X')
-      tictactoe.update_board(3, 'O')
+      tictactoe.update_board(3, 'X')
       tictactoe.update_board(4, 'O')
-      tictactoe.update_board(5, 'O')
+      tictactoe.update_board(5, 'X')
       tictactoe.update_board(6, 'X')
-      tictactoe.update_board(7, 'X')
-      tictactoe.update_board(8, 'O')
-      tictactoe.update_board(9, 'X')
-      expect(tictactoe.won?).to eq(false)
+      tictactoe.update_board(7, 'o')
+      tictactoe.update_board(8, 'o')
+      tictactoe.update_board(9, 'o')
+      expect(tictactoe.full?).to eq(true)
     end
-  end
 
-  describe 'full?' do
-    it 'should return true if the board is full' do
-      tictactoe.update_board(1, 'X')
-      tictactoe.update_board(2, 'X')
-      tictactoe.update_board(3, 'O')
-      tictactoe.update_board(4, 'O')
-      tictactoe.update_board(5, 'O')
-      tictactoe.update_board(6, 'X')
-      tictactoe.update_board(7, 'X')
+    it 'returns false when game is draw' do
+      tictactoe.update_board(1, 'O')
+      tictactoe.update_board(2, 'O')
+      tictactoe.update_board(3, 'X')
+      tictactoe.update_board(4, 'X')
+      tictactoe.update_board(5, 'X')
+      tictactoe.update_board(6, 'O')
+      tictactoe.update_board(7, 'O')
       tictactoe.update_board(8, 'O')
       tictactoe.update_board(9, 'X')
```

Comment:
The `draw` state is reached when 
1. The board is full
2. There is no winner.

Up There, you only tested if there is a winner and you forgot about the full state (even if you've already tested that method on section before) because the two conditions is needed.